modules/ossm-extensions-wasm-deploy.adoc

```diff
@@ -0,0 +1,33 @@
+[id="ossm-extensions-deploy_{context}"]
+= Deploying extensions
+
+{ProductName} extensions can be enabled using the `ServiceMeshExtension` resource. 
+
+.Procedure
+
+. Create the following example resource.
++
+.Example ServiceMeshExtension resource extension.yaml
+[source,yaml]
+----
+apiVersion: maistra.io/v1alpha1
+kind: ServiceMeshExtension
+metadata:
+  name: header-append
+  namespace: istio-system
+spec:
+  workloadSelector:
+    labels:
+      app: httpbin
+  config: test
+  image: quay.io/maistra-dev/header-append-filter:latest
```

Comment:
@neal-timpe The line 23 "image: quay.io/maistra-dev/header-append-filter:latest" is an image for OSSM 2.1 release.
For current 2.0.x release, we need to use "image: quay.io/maistra-dev/header-append-filter:2.0" . Could you add a note or update that ? Thanks.